apps/console/src/features/applications/components/forms/advanced-configurations-form.tsx

```diff
@@ -305,6 +305,7 @@ export const AdvancedConfigurationsForm: FunctionComponent<AdvancedConfiguration
             />
             {
                 (
+                    UIConfig?.showAPIBasedAuthentication &&
```

Comment:
This should go in to disabled features array.

@Achintha444 FYI

Reply:
Fixed with https://github.com/wso2/identity-apps/pull/4885/commits/2b56cd1fd7fc6ebe3bcc2fb316fa3e7b71d41240